docs/reference/commands/create-enrollment-token.asciidoc

```diff
@@ -57,3 +62,11 @@ The following command creates an enrollment token for enrolling an {es} node int
 ----
 bin/elasticsearch-create-enrollment-token -s node
 ----
+
+The following command creates an enrollment token for enrolling a kibana instance into a cluster, 
+while overriding the url where the local elasticsearch node is reachable:
```

Comment:
```suggestion
The following command creates an enrollment token for enrolling a {kib} instance into a cluster, 
and overrides the URL where the local {es} node is reachable:
```

Reply:
`and` feels off here ( and I mean "feels", I can't actually argue about it being wrong :). To me your suggestion reads as if the example command invocation does two things: 

1. It creates an enrollment token
2. It overrides the URL where the local elasticsearch node is reachable

which is not accurate. 
It does only 1. but 2 is something that temporarily happens - and only for the context of this command's execution - so that 1. can succeed. 

WDYT @lockewritesdocs  ?

Reply:
Ooooo, I see! In that case, let's explore options. Choose whichever one you like better, or let me know neither hits the mark.

This rendition is similar to your original text, but adds "temporarily" to distinguish the time constraint:

>The following command creates an enrollment token for enrolling a {kib} instance into a cluster, while temporarily overriding the URL where the local {es} node is reachable:

Another option dismisses the notion of the interaction between 1 and 2 entirely, and instead focuses on what the URL denotes:

>The following command creates an enrollment token for enrolling a {kib} instance into a cluster. The specified URL indicates where the local {es} node is reachable:

Reply:
Thanks for the discussion Adam. It actually makes me think that the original "is reachable" is misleading. Setting this option doesn't change at all where elasticsearch is reachable or listens to, but only where the CLI tool will talk to.
In my original version, there was a "while overriding the url where _the CLI tool thinks that_ the local elasticsearch node is reachable" implied there. 

With that context, do you have any suggestions on how to frame this ?

Reply:
Given that context, I think something like the following statement works:

>The following command creates an enrollment token for enrolling a {kib} instance into a cluster. The specified URL indicates where the `elasticsearch-create-enrollment-token` tool attempts to reach the local {es} node:

If we want to indicate more about "where the CLI tool thinks that the local ES node is reachable," then we might need further clarification of what happens if the tool doesn't find the node at the specified URL:

>The following command creates an enrollment token for enrolling a {kib} instance into a cluster. The specified URL indicates where the `elasticsearch-create-enrollment-token` tool attempts to reach the local {es} node. If the tool can't reach {es} at the specified port, it iterates through port numbers sequentially on the indicated host:

Caveat: I'm not certain that the last sentence here ☝️ is true.

Reply:
> Caveat: I'm not certain that the last sentence here point_up is true.

It doesn't work like this actually, 

> The following command creates an enrollment token for enrolling a {kib} instance into a cluster. The specified URL indicates where the elasticsearch-create-enrollment-token tool attempts to reach the local {es} node:

sounds good to me, let's keep this simple